recipe/meta.yaml

```
@@ -10,19 +10,21 @@ source:
   sha256: 0dbd14a8aef6636764f88a8fd1fcc9a91921e5c50356e6aab251782f264ae960
 
 build:
```

```suggestion
build:
  skip: true  # [py<38]
```
Might be worth explicitly specifying compatible Python versions. See [pyproject.toml.](https://github.com/jupyterlab/hatch-jupyter-builder/blob/74d37d9854668fea5c51b57622aa12fc72f0a61c/pyproject.toml#L10)